app/frontend/styles/components/search_panel.scss

```diff
@@ -45,7 +43,6 @@
 
   .location-radius-select {
     display: inline-block;
-    float: right;
```

Comment:
this makes so sense alongside inline-block so removed it

Reply:
Nevertheless, we need the top image not the bottom one. This branch regresses to the bottom image